bahmnicore-omod/src/main/java/org/bahmni/module/bahmnicore/web/v1_0/controller/BahmniVisitLocationController.java

```diff
@@ -32,4 +34,25 @@ public HashMap<String, String> getVisitLocationInfo(@PathVariable("loginLocation
         return visitLocation;
     }
 
+    @RequestMapping(method = RequestMethod.GET, value = "/facility/{locationUuid}")
+    @ResponseBody
+    public HashMap<String, String> getFacilityVisitLocationInfo(@PathVariable("locationUuid") String locationUuid ) {
```

Comment:
Can you add tests for different contexts please?

Reply:
added test cases